### DISCUSSION TITLE
Table data extraction from pdf

### DISCUSSION THREAD

**Discussion by Mano2731995** [pymupdf/PyMuPDF#2187] [Looking for help]
Hai jorj

Is there any possible to extract the table by using pymupdf? 

I need only the table inside the datas so I need to extract table column and row wise. 

Kindly suggest to proceed further.

**Comment by JorjMcKie** [comment]
The typically greatest problem is how to find the position of the table on the page. For this knowledge from outside (i.e., **_from you_**) is required.
Maybe there are keywords helping to find the location, by searching for them.

The table analysis itself can be complex, too.
* If there are gridlines, we have a solution [here](https://github.com/pymupdf/PyMuPDF-Utilities/blob/master/table-analysis/table-with-gridlines.py).
* Otherwise there is significant programming effort ahead to find column and row borders.

So please be more specific as per your situation.
